src/vs/workbench/contrib/relauncher/browser/relauncher.contribution.ts

```diff
@@ -258,6 +261,50 @@ export class WorkspaceChangeExtHostRelauncher extends Disposable implements IWor
 	}
 }
 
+export class RuntimeArgumentsChangeRelauncher extends Disposable implements IWorkbenchContribution {
+
+	private readonly disableChromiumSandbox = new ChangeObserver('boolean');
```

Comment:
We probably also need the initial state, no? What if the user has this already configured?